src/qibo/models/error_mitigation.py

```diff
@@ -809,12 +813,12 @@ def error_sensitive_circuit(circuit, observable, backend=None):
     for i in range(num_qubits):
         observable_i = pauli_gates[observable_pauli[i]]
         random_init = pauli_gates["I"]
-        while np.any(abs(observable_i - pauli_gates["Z"]) > 1e-5) and np.any(
-            abs(observable_i - pauli_gates["I"]) > 1e-5
-        ):
+        while backend.np.any(
+            backend.np.abs(observable_i - pauli_gates["Z"]) > 1e-5
+        ) and backend.np.any(abs(observable_i - pauli_gates["I"]) > 1e-5):
```

Comment:
is the `np.any` even needed here or can the built-in `any` from `python` be used?